src/stories/Filters.js

```diff
@@ -7,7 +7,9 @@ import PokeDex from './assets/pokedex'
 
 const sieve = SchemaSieve()
 
-const Container = styled.div`margin: 30px;`
+const Container = styled.div`
+  margin: 30px;
+`
 
 const StyledTable = styled.table`
```

Comment:
We should add an issue to add a table component as well.

Reply:
Agreed! https://github.com/resin-io-modules/resin-components/issues/157